src/commands/report.py

```diff
@@ -81,7 +81,7 @@ def validate_verbose_test_failure_reporting_ticket_limit(
     "--firewatch-config-path",
     help="The path to the firewatch configuration file",
     required=False,
-    type=click.Path(exists=True),
```

Comment:
I wonder if, rather than removing this check, we add a function that checks if the path provided is a URL or a path. If it is a URL, set this to false. That way, if someone does pass a file path, it will still check if it exists.

Reply:
@calebevans We do already have [read_base_config_file](https://github.com/RedHatQE/firewatch/blob/main/src/objects/configuration.py#L15), do you think we need to do anything else?

Reply:
@oharan2 good point, forgot about that. I think that is probably good then.